src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

```suggestion
  min-width: 1024px;
```
maybe you meant _min-width_?